examples/python/mapping/full_pipeline.py

```diff
@@ -0,0 +1,122 @@
+import os
+import csv
+from PIL import Image as PilImage
+import numpy as np
+import pywavemap as wave
+
+# Parameters
+home_dir = os.path.expanduser('~')
+measurement_dir = os.path.join(home_dir,
+                               "data/panoptic_mapping/flat_dataset/run2")
+output_map_path = os.path.join(home_dir, "your_map.wvmp")
+
+# Create a map
+your_map = wave.Map.create({
+    "type": "hashed_chunked_wavelet_octree",
+    "min_cell_width": {
+        "meters": 0.05
+    }
+})
+
+# Create a measurement integration pipeline
+pipeline = wave.Pipeline(your_map)
+# Add map operations
+pipeline.addOperation({
```

Comment:
This naming style would be against PEP8 convention where lower case with space is the desired standard (https://peps.python.org/pep-0008/#method-names-and-instance-variables)